sdk/nodejs/tests/automation/localWorkspace.spec.ts

```diff
@@ -1289,6 +1398,50 @@ describe("LocalWorkspace", () => {
     });
 });
 
+/**
+ * Augments the provided {@link LocalWorkspaceOptions} so that they reference a
+ * file backend backed by a temporary directory. The temporary directory (and
+ * any contents) will be cleaned up on the test suite's exit.
+ */
+function withTemporaryFileBackend(opts?: LocalWorkspaceOptions): LocalWorkspaceOptions {
+    const tmpDir = tmp.dirSync({
+        tmpdir: "/dev/shm",
+        prefix: "nodejs-tests-automation-",
```

Comment:
```suggestion
        prefix: "nodejs-tests-automation-",
        template: "XXXXXXXX",
```

The docs say `template` has no default, so maybe this is necessary?  Not sure what no default there means. https://www.npmjs.com/package/tmp